project_event/tests/common.py

```diff
@@ -178,3 +178,41 @@ def setUp(self):
             'room_id': self.room_1.id,
             'notes': 'Some Task Notes ...',
         })
+        self.task_3 = self.Tasks.create({
+            'name': 'Sample task3',
+            'activity_task_type': 'activity',
+            'partner_id': self.project_1.partner_id.id,
+            'room_id': self.room_1.id,
+            'date_start': fields.Datetime.to_string(datetime.today()),
+            'date_end': fields.Datetime.to_string(datetime.today() +
+                                                  timedelta(hours=4)),
+            'real_date_start': fields.Datetime.to_string(datetime.today()),
+            'real_date_end': fields.Datetime.to_string(datetime.today()),
+            'total_time': " ",
+        })
+        self.task_4 = self.Tasks.create({
+            'name': 'Sample task4',
+            'activity_task_type': 'activity',
```

Comment:
task or activity?

Reply:
Check if it is a task or activity, same comment for the above

Reply:
You should put the fields that are only used in one test file, in that test file. Common is for fields that will be referenced in more than one test file